### ISSUE TITLE
Lag while someone is requesting songs (Raspberry pi 3)

### ISSUE THREAD

**Issue by Lawlietnao** [Just-Some-Bots/MusicBot#1319]
The Current song will be choppy and laggy while someone is requesting songs(!play). I'm using Raspberry Pi 3 Model B, not sure if that's the problem. By the way, big thanks to you guys for what you have done to all the music bot users!

**Comment by AnnsAnns** [comment]
That is not a bug, it is a normal behavior of a slowed down internet connection

**Comment by Lawlietnao** [comment]
@tom-GER 
Dear tom thank you so much for your reply. However, I used to hold this bot on my desktop and it worked like a charm. I'm using a wired connection on my RBP 3, so I can confirm that it's not caused by the connection issue, and my CPU usage maintains around 10~30 while searching songs, so I think it's also not caused by hardware.

Here's my speed test result.
http://i.imgur.com/DMerQ0d.png

In addition, I've found that if you accidentally play a live video, the bot will crash and show this error.
http://i.imgur.com/wYXH7kO.png

Not sure if this is just a bug or only me, I've tried to reinstall ffmpeg twice tho.
